---a/net/minecraft/world/biome/BiomeGenJungle.java
+++b/net/minecraft/world/biome/BiomeGenJungle.java
@@ -1,6 +1,7 @@
 package net.minecraft.world.biome;
 
 import java.util.Random;
+
 import net.minecraft.block.BlockPlanks;
 import net.minecraft.block.BlockTallGrass;
 import net.minecraft.entity.passive.EntityChicken;
@@ -16,48 +16,42 @@
 import net.minecraft.world.gen.feature.WorldGenVines;
 import net.minecraft.world.gen.feature.WorldGenerator;
 
-public class BiomeGenJungle extends BiomeGenBase
-{
+public class BiomeGenJungle extends BiomeGenBase {
+
     private boolean field_150614_aC;
     private static final String __OBFID = "CL_00000175";
 
-    public BiomeGenJungle(int p_i45379_1_, boolean p_i45379_2_)
-    {
+    public BiomeGenJungle(int p_i45379_1_, boolean p_i45379_2_) {
         super(p_i45379_1_);
         this.field_150614_aC = p_i45379_2_;
 
-        if (p_i45379_2_)
-        {
+        if (p_i45379_2_) {
             this.theBiomeDecorator.treesPerChunk = 2;
-        }
-        else
-        {
+        } else {
             this.theBiomeDecorator.treesPerChunk = 50;
         }
 
         this.theBiomeDecorator.grassPerChunk = 25;
         this.theBiomeDecorator.flowersPerChunk = 4;
 
-        if (!p_i45379_2_)
-        {
+        if (!p_i45379_2_) {
             this.spawnableMonsterList.add(new BiomeGenBase.SpawnListEntry(EntityOcelot.class, 2, 1, 1));
         }
 
         this.spawnableCreatureList.add(new BiomeGenBase.SpawnListEntry(EntityChicken.class, 10, 4, 4));
     }
 
-    public WorldGenAbstractTree genBigTreeChance(Random p_150567_1_)
-    {
-        return (WorldGenAbstractTree)(p_150567_1_.nextInt(10) == 0 ? this.worldGeneratorBigTree : (p_150567_1_.nextInt(2) == 0 ? new WorldGenShrub(BlockPlanks.EnumType.JUNGLE.getMetadata(), BlockPlanks.EnumType.OAK.getMetadata()) : (!this.field_150614_aC && p_150567_1_.nextInt(3) == 0 ? new WorldGenMegaJungle(false, 10, 20, BlockPlanks.EnumType.JUNGLE.getMetadata(), BlockPlanks.EnumType.JUNGLE.getMetadata()) : new WorldGenTrees(false, 4 + p_150567_1_.nextInt(7), BlockPlanks.EnumType.JUNGLE.getMetadata(), BlockPlanks.EnumType.JUNGLE.getMetadata(), true))));
+    public WorldGenAbstractTree genBigTreeChance(Random p_150567_1_) {
+        return (WorldGenAbstractTree) (p_150567_1_.nextInt(10) == 0 ? this.worldGeneratorBigTree : (p_150567_1_.nextInt(2) == 0 ? new WorldGenShrub(BlockPlanks.EnumType.JUNGLE.getMetadata(), BlockPlanks.EnumType.OAK.getMetadata()) : (!this.field_150614_aC && p_150567_1_.nextInt(3) == 0 ? new WorldGenMegaJungle(false, 10, 20, BlockPlanks.EnumType.JUNGLE.getMetadata(), BlockPlanks.EnumType.JUNGLE.getMetadata()) : new WorldGenTrees(false, 4 + p_150567_1_.nextInt(7), BlockPlanks.EnumType.JUNGLE
+                .getMetadata(), BlockPlanks.EnumType.JUNGLE.getMetadata(), true))));
     }
 
-    public WorldGenerator getRandomWorldGenForGrass(Random p_76730_1_)
-    {
+    public WorldGenerator getRandomWorldGenForGrass(Random p_76730_1_) {
         return p_76730_1_.nextInt(4) == 0 ? new WorldGenTallGrass(BlockTallGrass.EnumType.FERN) : new WorldGenTallGrass(BlockTallGrass.EnumType.GRASS);
     }
 
-    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_)
-    {
+    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_) {
         super.decorate(worldIn, p_180624_2_, p_180624_3_);
         int i = p_180624_2_.nextInt(16) + 8;
         int j = p_180624_2_.nextInt(16) + 8;
@@ -65,13 +65,11 @@
         (new WorldGenMelon()).generate(worldIn, p_180624_2_, p_180624_3_.add(i, k, j));
         WorldGenVines worldgenvines = new WorldGenVines();
 
-        for (j = 0; j < 50; ++j)
-        {
+        for (j = 0; j < 50; ++j) {
             k = p_180624_2_.nextInt(16) + 8;
             boolean flag = true;
             int l = p_180624_2_.nextInt(16) + 8;
             worldgenvines.generate(worldIn, p_180624_2_, p_180624_3_.add(k, 128, l));
         }
     }
-}
+}
